python/dgl/nn/pytorch/explain/shapley.py

```diff
@@ -0,0 +1,159 @@
+"""Utility functions for SubgraphX"""
```

Comment:
Maybe merge this file with `subgraphxexplainer.py`

Reply:
Done.